templates/slider.jsx

```diff
@@ -32,7 +32,9 @@ export default function Slider (props) {
     _showScale,
     _showNumber,
     _showScaleNumbers,
-    _showScaleIndicator
+    _showScaleIndicator,
+    scaleStepPrefix,
+    scaleStepSuffix
```

Comment:
Do you need to set defaults for these?
```suggestion
    scaleStepPrefix = '',
    scaleStepSuffix = ''
```

Reply:
@oliverfoster I don't believe this is necessary as it will resolve to a falsy value in JSX and React will ignore it / not print anything. Just tested this without setting a prefix/suffix, and it seems fine. From a best practices perspective, would it be better to set default values anyway?

Reply:
Awesome, good to know. I can never remember which of the templating languages prints "undefined". Thanks for checking.